toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -139,10 +156,11 @@ func resolveGraphNodes(dependencyGraph *pkggraph.PkgGraph, inputSummaryFile, out
 		timestamp.StartEvent("clone graph", nil)
 
 		for _, n := range unresolvedNodes {
-			resolveErr := resolveSingleNode(cloner, n, toolchainPackages, fetchedPackages, prebuiltPackages, *outDir)
+			resolveErr := resolveSingleNode(cloner, n, downloadDependencies, toolchainPackages, fetchedPackages, prebuiltPackages, *outDir)
 			// Failing to clone a dependency should not halt a build.
 			// The build should continue and attempt best effort to build as many packages as possible.
 			if resolveErr != nil {
+				logger.Log.Warn(resolveErr)
```

Comment:
>logger.Log.Warn(resolveErr) [](http://example.com/codeflow?start=4&length=27)

One more request: could we add a bit of comment to this warning, so it's clearer where it's coming from and what was happening at the time?